qemu/tests/cfg/blockdev_commit.cfg

```diff
@@ -42,9 +42,9 @@
     variants:
         - @one_data_disk:
         - multi_data_disks:
-            image_size_data1 = 300M
-            image_name_data1 = data1
-            snapshot_tags_data1 = sn11 sn21 sn31 sn41
+            image_size_data2 = 2G
```

Comment:
why we want a data2 here?

Reply:
For most cases, data/data1 is used as the first data disk, and data2 is used as the second data disk. To make them unified, we re-define the second data disk here.